Cython/Compiler/Options.py

```diff
@@ -423,7 +423,7 @@ def parse_directive_list(s, relaxed_bool=False, ignore_unknown=False,
         item = item.strip()
         if not item:
             continue
-        if not '=' in item:
+        if '=' not in item:
```

Comment:
It's a bit sad that these changes slipped in. Probably a commit hook on your side or so.
While I think that cleaning them up is a good thing, it should better not happen as part of an unrelated commit.